### PR TITLE
F/wpc 408 navbar collapse

### PR DESCRIPTION
**Pull request by camilleory**
Sidebar collapse button on the bottom right removed and placed on the top right, styled changed to be better visible. 
Sidebar color changed from white to default.

### CODE REVIEW

File: packages/editor/src/client/base.tsx [hunk 1]
```diff
@@ -75,14 +75,31 @@ export function Base({children}: BaseProps) {
...
+              <IconButton
+                style={{
+                  position: 'absolute',
+                  top: '5vh',
+                  left: isExpanded ? 260 : 56,
+                  transform: 'translate(-50%)',
+                  zIndex: 100
+                }}
+                className="collapse-nav-btn"
+                appearance="primary"
```
Comment (penk85) [inline packages/editor/src/client/base.tsx:96]: Personal preference: I think it's okay with a subtle style now that it's higher up and more visible

**Review by penk85 — DISMISSED** [review]
Looks great! :+1:

**Review by HanaAyman — DISMISSED** [review]

**Review by bishoymelek — APPROVED** [review]
Screenshot from 2021-06-17 21-11-04
Amazing work @camilleory :tada:

**Review by roosnic1 — APPROVED** [review]
@camilleory looks very good. I just made a tiny CSS change that keeps the icon a little bit longer on the screen and therefor it has less flickering.

**Review by bishoymelek — APPROVED** [review]

**Pull request merged**